### ISSUE TITLE
Wrapping and unwrapping DataNewtype blows up script budget

### ISSUE THREAD

**Issue by colll78** [Plutonomicon/plutarch-plutus#764]
https://github.com/Plutonomicon/plutarch-plutus/blob/429d1b0292d87650a7beb1351612629434a0903f/plutarch-ledger-api/src/Plutarch/LedgerApi/V1/Time.hs#L67-L96

This module is bricked. Switched from `PInteger` to `PPosixTime` and my benchmarks had a huge regression. The issue is that all the operations wrap and unwrap the data, this results in a ton of `iData` and `unIData` builtin calls, because each addition unwrapped each integer from data and then wraps the result. Instead we should have two definitions `PosixTimeData` and `PosixTime` or anything else so that this can be avoided and we can just keep it in the non-data encoded format until we want to go back to data encoding.

**Comment by SeungheonOh** [comment]
Yes, I'm aware of this issue. We are going to remove `PDataNewtype` which was a big mistake
